build.sh

```diff
@@ -43,7 +43,7 @@ do_build_native() {
 }
 
 do_build_docker() {
-    if ! dpkg --print-architecture | grep -q 'amd64'; then
+    if [[ -f $( which dpkg ) && $( dpkg --print-architecture | head -1 ) != "amd64" ]]; then
```

Comment:
this was actually fine without checking for `dpkg` presence, as it's absence is an error which, when negated, would lead to erroring out :)

Reply:
I don't have an ARM system to test on. Please suggest a working version or merge as-is and let someone affected work it out.

Reply:
The above diff has no impact of the code flow, except the new version runs more commands (thus marginally slower).
Readability should be the main concern here, but not a blocker either way imho.
```bash
$ [[ -f $( which dpkg ) && $( dpkg --print-architecture | head -1 ) != "amd64" ]]; echo $?
1
$ ! dpkg --print-architecture | grep -q 'amd64'; echo $?
1
$ ! foodpkg --print-architecture | grep -q 'amd64'; echo $?
foodpkg: command not found
0
$ ! dpkg --print-architecture | grep -q 'armhf'; echo $?
0
$ ! foodpkg --print-architecture | grep -q 'armhf'; echo $?
foodpkg: command not found
0
```

Reply:
Yea this little thing is going to be a big tweak later on, but I just don't have any machines other than amd64 Debian 10 to test on ;-) I'm sure there's a nice clean way to do it.